CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Node.js Agent for Google Cloud Debug ChangeLog
 
+## 2017-07-17, Version 2.1.3 (Beta), @ofrobots
+
+This module is now in Beta. This release (re)-fixes the naming of debug targets on GKE (#308) along with a fuzzy search for sourcemaps (#306).
+
+### Commits
+* [[`eff37c20f5`](https://github.com/Google Inc./debug-agent/commit/eff37c20f5)] - beta (#310) (Ali Ijaz Sheikh)
```

Comment:
`Google Inc. => GoogleCloudPlatform` here and elsewhere and `debug-agent => cloud-debug-nodejs` here an elsewhere.